store/memory_test.go

```diff
@@ -100,3 +102,56 @@ func TestExpiration(t *testing.T) {
 	require.Error(t, err)
 
 }
+
+func TestLatency(t *testing.T) {
+	t.Parallel()
+
+	putLatency := 1 * time.Second
+	getLatency := 1 * time.Second
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	kzgConfig := &kzg.KzgConfig{
+		G1Path:          "../resources/g1.point",
+		G2PowerOf2Path:  "../resources/g2.point.powerOf2",
+		CacheDir:        "../resources/SRSTables",
+		SRSOrder:        3000,
+		SRSNumberToLoad: 3000,
+		NumWorker:       uint64(runtime.GOMAXPROCS(0)),
+	}
+
+	cfg := &verify.Config{
+		Verify:    false,
+		KzgConfig: kzgConfig,
+	}
+
+	verifier, err := verify.NewVerifier(cfg, nil)
+	require.NoError(t, err)
+
+	ms, err := NewMemStore(
+		ctx,
+		verifier,
+		log.New(),
+		1024*1024*2,
+		time.Millisecond*10,
+		putLatency, getLatency,
+	)
+
+	require.NoError(t, err)
+
+	preimage := []byte(testPreimage)
+	timeBeforePut := time.Now()
+	key, err := ms.Put(ctx, preimage)
+	require.NoError(t, err)
+	require.GreaterOrEqual(t, time.Since(timeBeforePut), putLatency)
+
+	// sleep 1 second and verify that older blob entries are removed
+	time.Sleep(time.Second * 1)
```

Comment:
is this sleep necessary for this test case?

Reply:
good catch, [edcf800](https://github.com/Layr-Labs/eigenda-proxy/pull/114/commits/edcf800dd62a49d6f2c8dd6a11fa4ab6ebb9efb0)